src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/PersonDetailsService.kt

```diff
@@ -36,8 +37,8 @@ class PersonDetailsService(
         crn = crn
       ),
       currentAddress = CurrentAddress(
-        line1 = formatTwoWordField(addressNumber, buildingName),
-        line2 = activeAddress?.district ?: "",
+        line1 = formatTwoWordField(addressNumber, streetName),
+        line2 = formatTwoWordField(buildingName, activeAddress?.district ?: ""),
```

Comment:
Would building name be part of line 1, if present? ie in front of the street name

Reply:
As it is, the building name will always be in line 2. Hmm, which do you think is best Jon? I'll incline to your suggestion :)

Reply:
Done, using [building name] [building number] [street name], verified by Ale :)